packages/auth/src/providers/cognito/apis/signInWithRedirect.ts

```diff
@@ -110,6 +111,11 @@ const oauthSignIn = async ({
 	// TODO(v6): use URL object instead
 	const oAuthUrl = `https://${domain}/oauth2/authorize?${queryString}`;
 
+	// this will be effective only on the following scenarios:
+	// 1. the user cancels the OAuth flow on web, and
```

Comment:
Nit: Should this be narrowed down to the back button? E.g.
```suggestion
	// 1. the user cancels the OAuth flow on web via back button, and
```